Formula/p/podman-tui.rb

```diff
@@ -0,0 +1,26 @@
+class PodmanTui < Formula
+  desc "Podman Terminal User Interface"
+  homepage "https://github.com/containers/podman-tui"
+  url "https://github.com/containers/podman-tui.git",
+      tag:      "v0.17.0",
+      revision: "b4350927babdaa35fbf50424190683aeb9f5f66f"
```

Comment:
I dont see the need to use git checkout?

Reply:
Thanks for review, updated and url now pointing to tar.gz package